src/overlays/actors/ovl_Item_Etcetera/z_item_etcetera.h

```diff
@@ -5,12 +5,20 @@
 
 struct ItemEtcetera;
 
+#define ITEMETCETERA_GET_FF(thisx) ((thisx)->params & 0xFF)
+#define ITEMETCETERA_GET_TREASURE(thisx) (((thisx)->params >> 8) & 0x1F)
+
 typedef void (*ItemEtceteraActionFunc)(struct ItemEtcetera*, GlobalContext*);
 
 typedef struct ItemEtcetera {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x10];
-    /* 0x0154 */ ItemEtceteraActionFunc actionFunc;
+    /* 0x0144 */ ItemEtceteraActionFunc actionFunc;
+    /* 0x148 */ s16 unk_148;
+    /* 0x14A */ s16 unk_14A;
+    /* 0x14C */ u8 unk_14C;
+    /* 0x14D */ char pad_14D[3];
```

Comment:
```suggestion
    /* 0x14D */ UNK_TYPE1 pad_14D[3];
```

Reply:
Should be possible to remove it altogether given the alignment of the next member.

Reply:
how?

Reply:
See tutorial section on struct padding.

Reply:
i know, but still:

typedef struct ItemEtcetera {
    /* 0x0000 */ Actor actor;
    /* 0x0144 */ ItemEtceteraActionFunc funcSetup;
    /* 0x148 */ s16 unk_148;
    /* 0x14A */ s16 unk_14A;
    /* 0x14C */ u8 unk_14C;
    /* 0x14D */ char pad_14D[3];
    /* 0x0150 */ ActorFunc drawFunc;
    /* 0x0154 */ ItemEtceteraActionFunc actionFunc;
} ItemEtcetera; // size = 0x158

padding lines up perfectly fine for me. If i take it out, ItemEtcetera would end up being size = 0x155, and that doesn't match given sizeof(ItemEtcetera) inside initvars.

Reply:
mmm, you are right, it matches somehow. I don't understand why since i've had issues with that in other occasions.